[SYTO-1096] Support the beta rollout for mortar v3

asabaylus opened the pull request:
```
$ git checkout SYTO-1096
$ gulp serve
```

Validate the following ...

1 add a version designation to the landing page
2 for the future add analytics to the documentation
3 correct v4 to v3 on the landing page
4 add badges to the sharing
5 update the css link
6 Add to the important note "If you use in development feature you should expect to see errors."

arieldelfor commented on src/site/index.html:
```diff
@@ -28,7 +32,7 @@ <h1 class="mt3_h1 intro-txt">Welcome to Mortar</h1>
 
   <p class="mt3_subh4 mt3_row-gut-2">
-    Mortar v4.0 is pre-release software. Some features will be deprecated and removed from Mortar prior to general release. Some features are still under development. Please avoid using any feature with the <span class="badge">In Development</span> badge.
+    Mortar v3.0 is pre-release software. Some features will be deprecated and removed from Mortar prior to general release. Some features are still under development. Please avoid using any feature with the <span class="badge">In Development</span> badge. If you use features "In Development" you should expect to see errors.
```
> If you use features "In Development" you should expect to see errors

not sure why i like this kind of agreement 🎉

arieldelfor commented on src/site/index.html:
```diff
@@ -6,13 +6,17 @@
 
       <code class="mt3_bgcolor--white mt3_row-gut-2 codemarkup">
-        	&lt;link href='https://www.nationalgeographic.com/styleguide/mortar_2-0-0.css' rel='stylesheet'&gt;
+        	&lt;link href='https://style-guide.nationalgeographic.com/beta/styles/mortar.min.css' rel='stylesheet'&gt;
```
SSL/TLS seems not working at that URL. I guess it should be HTTP instead ?

pmourelle commented:
LGTM

krlospelaez commented:
PR Ok

asabaylus commented:
You should feel free to change this PR as you see fit. I haven't got the time to to do it. So if ya want to change do it ?

Sent from my iPad

On Oct 24, 2016, at 2:46 PM, Ariel <notifications@github.com<mailto:notifications@github.com>> wrote:


@arieldelfor commented on this pull request.

________________________________

In src/site/index.html<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_natgeo_mortar_pull_379-23pullrequestreview-2D5512825&d=DQMFaQ&c=uw6TLu4hwhHdiGJOgwcWD4AjKQx6zvFcGEsbfiY9-EI&r=p56RlrbJtdI-z284QW3JOFsjHh6yui35WEl8Uh5yRMQ&m=-dhBTvHgxmpMaXQ1kGnGXCJwgAK0anEnfqajMSXZwgY&s=zzZQA5tCjUzLxpAjwgGFbvWDCYSMR31JrQKkqmsmtO8&e=>:

> @@ -28,7 +32,7 @@ <h1 class="mt3_h1 intro-txt">Welcome to Mortar</h1>
   <h3 class="mt3_h3">Important Note</h3>

   <p class="mt3_subh4 mt3_row-gut-2">
-    Mortar v4.0 is pre-release software. Some features will be deprecated and removed from Mortar prior to general release. Some features are still under development. Please avoid using any feature with the <span class="badge">In Development</span> badge.
+    Mortar v3.0 is pre-release software. Some features will be deprecated and removed from Mortar prior to general release. Some features are still under development. Please avoid using any feature with the <span class="badge">In Development</span> badge. If you use features "In Development" you should expect to see errors.


If you use features "In Development" you should expect to see errors

not sure why i like this kind of agreement ?

________________________________

In src/site/index.html<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_natgeo_mortar_pull_379-23pullrequestreview-2D5512825&d=DQMFaQ&c=uw6TLu4hwhHdiGJOgwcWD4AjKQx6zvFcGEsbfiY9-EI&r=p56RlrbJtdI-z284QW3JOFsjHh6yui35WEl8Uh5yRMQ&m=-dhBTvHgxmpMaXQ1kGnGXCJwgAK0anEnfqajMSXZwgY&s=zzZQA5tCjUzLxpAjwgGFbvWDCYSMR31JrQKkqmsmtO8&e=>:

>
       <p class="mt3_subh1 intro-txt mt3_row-gut-2">
         Mortar is the official Style Guide for National Geographic&apos;s digital products. To use Mortar, add the following line of code into the header of your project:
       </p>

       <code class="mt3_bgcolor--white mt3_row-gut-2 codemarkup">
-               &lt;link href='https://www.nationalgeographic.com/styleguide/mortar_2-0-0.css' rel='stylesheet'&gt;
+               &lt;link href='https://style-guide.nationalgeographic.com/beta/styles/mortar.min.css' rel='stylesheet'&gt;


SSL/TLS seems not working at that URL. I guess it should be HTTP instead ?

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_natgeo_mortar_pull_379-23pullrequestreview-2D5512825&d=DQMFaQ&c=uw6TLu4hwhHdiGJOgwcWD4AjKQx6zvFcGEsbfiY9-EI&r=p56RlrbJtdI-z284QW3JOFsjHh6yui35WEl8Uh5yRMQ&m=-dhBTvHgxmpMaXQ1kGnGXCJwgAK0anEnfqajMSXZwgY&s=zzZQA5tCjUzLxpAjwgGFbvWDCYSMR31JrQKkqmsmtO8&e=>, or mute the thread<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_AAa8txmdoNgCfkuhsQNwwLEocvAJuwo3ks5q3P0SgaJpZM4KfHFk&d=DQMFaQ&c=uw6TLu4hwhHdiGJOgwcWD4AjKQx6zvFcGEsbfiY9-EI&r=p56RlrbJtdI-z284QW3JOFsjHh6yui35WEl8Uh5yRMQ&m=-dhBTvHgxmpMaXQ1kGnGXCJwgAK0anEnfqajMSXZwgY&s=TcoojWTP2nAtDlRwleXA8xcaHAYOg_Wf2BJmrqi9Qbc&e=>.
